src/components/ProfileInformation/ProfileInformation.tsx

```diff
@@ -112,7 +112,9 @@ const ProfileInformation = (props: Props) => {
           {!isBlocked && (
             // The class name is needed to avoid the display block of the div. The div is just for testing purposes
             <div data-testid={actionsForNonBlockedTestId} className={styles.displayContents}>
-              {shouldShowFriendsButton ? <FriendshipButton className={styles.friendsButton} friendAddress={profileAddress} /> : null}
+              {shouldShowFriendsButton ? (
```

Comment:
Both `bool && <component />` and `bool ? <component /> : null` are used here, should we normalize it to one of the two?

Reply:
The standard in our dApps is to use the ternary notation, there might be some places were `&&` was used but we can correct them in another PR, what do you think about it?